bsp/boards/openmote-b/board.c

```diff
@@ -133,17 +153,98 @@ bool board_timer_expired(uint32_t future) {
     uint32_t current;
     int32_t remaining;
 
+    /* Get current time */
     current = TimerValueGet(GPTIMER2_BASE, GPTIMER_A) >> 5;
 
+    /* Calculate remaining time */
     remaining = (int32_t) (future - current);
-    
+
+    /* Return if timer has expired */
     if (remaining > 0) {
         return false;
     } else {
         return true;
     }
 }
 
+//==== bootstrapping slot info lookup table
+void board_init_slot_vars(void){
+    //10ms slot
```

Comment:
we don't care about anything than 3 settings, and 40ms timeslots.

Reply:
the 10ms and 20ms are used in other board, should I definitely remove them? @changtengfei @TimothyClaeys